README.md

```diff
@@ -9,24 +9,44 @@ This site is written using [Jekyll](https://jekyllrb.com/), a static site genera
 ## Local Development
 
 #### Prerequisites:
+
 1. [Ruby](https://www.ruby-lang.org/en/documentation/installation/)
 2. [Bundler](https://bundler.io/) (`gem install bundler`)
 
 #### Instructions
+
 1. `bundle install`
 2. Add necessary environment variables (they MUST be defined!!!):
-	* `export CSHPUBSITE_ASSETS_URL="https://assets.csh.rit.edu/pubsite"`
-	* `export CSHPUBSITE_S3_URL="https://s3.csh.rit.edu"`
+   - `export CSHPUBSITE_ASSETS_URL="https://assets.csh.rit.edu/pubsite"`
+   - `export CSHPUBSITE_S3_URL="https://s3.csh.rit.edu"`
 3. `bundle exec jekyll serve`
 
 The site should now serve locally at `localhost:4000`, and auto-build when you change/create files in the repo!
 
+### Running in a container
+
+To run this site in a container, make sure you have either [Podman](https://podman.io) or [Docker](https://docker.com) installed.
+
+First, build the container with the following command
+
+```
+podman build -t public-site .
+```
+
+Then, run the container with the following command
+
+```
+podman run -p 8080:80 public-site:latest
```

Comment:
Actually, can you make this `podman run -p 8080:80 -it public-site`? That way, people don't get stuck when ctrl-c doesn't work.